pixeltable/type_system.py

```diff
@@ -1455,3 +1442,32 @@ def as_col_type(cls, nullable: bool) -> ColumnType:
 
 
 ALL_PIXELTABLE_TYPES = (String, Bool, Int, Float, Timestamp, Json, Array, Image, Video, Audio, Document)
+
+
+def column_type_from_np_dtype(dtype: np.dtype, nullable: bool) -> ColumnType | None:
```

Comment:
I'd suggest making this a `@classmethod` of `ColumnType` (`ColumnType.from_np_dtype`)